angular-ngc/tools/BUILD.bazel

```diff
@@ -34,6 +35,19 @@ ts_project(
 exports_files(
     [
         "karma.conf.js",
+        "test-setup.ts",
     ],
     visibility = ["//visibility:public"],
 )
+
+esbuild(
+    name = "prebuild_test_bootstrap",
+    testonly = 1,
+    entry_points = ["//tools:test-bootstrap.js"],
```

Comment:
Why are we using entry_point*s* + output_dir + splitting? Don't we want a single file?

Reply:
yeah, you are right, I removed the dir and splitting options.

Reply:
and switch to `entry_point`. Think it will only output a single file instead of directory

Reply:
Got it, done.